spec/models/facility_spec.rb

```diff
@@ -7,22 +7,177 @@
     it { should have_many(:blood_sugars).through(:encounters).source(:blood_sugars) }
     it { should have_many(:prescription_drugs) }
     it { should have_many(:patients).through(:encounters) }
-    it { should have_many(:hypertension_patients).through(:blood_pressures) }
     it { should have_many(:appointments) }
 
     it { should have_many(:registered_patients).class_name('Patient').with_foreign_key('registration_facility_id') }
 
-    it 'has distinct patients' do
-      facility = create(:facility)
-      patient = create(:patient)
-      blood_pressures = create_list(:blood_pressure, 2, facility: facility, patient: patient)
-      blood_sugars = create_list(:blood_sugar, 2, facility: facility, patient: patient)
-      (blood_pressures + blood_sugars).each { |record| create(:encounter, :with_observables, patient: patient, observable: record, facility: facility) }
-      expect(facility.patients.count).to eq(1)
+    context 'patients' do
+      it 'has distinct patients' do
+        facility =
```

Comment:
Whats with the formatting here? Why are these multiline?

Reply:
Just did it for readability, not an important change. I'll go and revert it.